src/app.tsx

```diff
@@ -73,9 +74,11 @@ const Application = () => {
     };
 
     useEffect(() => {
-        // cockpit doesn't provide type for addEventListener so it needs to be ignored
-        // @ts-expect-error addEventListener method is not added to type definitions yet
-        superuser.addEventListener("changed", () => { setAuthenticated(superuser.allowed) });
+    // cockpit doesn't provide type for addEventListener so it needs to be ignored
+    // @ts-expect-error addEventListener method is not added to type definitions yet
+        superuser.addEventListener("changed", () => {
+            setAuthenticated(superuser.allowed);
+        });
```

Comment:
Please don't add random formatting changes to PRs. It just adds noise

Reply:
I didn't if the change came through it was cause of npm run eslint:fix

Reply:
I didn't make the change it was cause by `npm run eslint:fix`

Reply:
do we want to ignore what eslint wants and stick to what we prefer?

Reply:
I've moved the change into its own commit, It will keep coming back any time someone runs `eslint:fix` on the codebase, might as well include it

Reply:
I doubt we are not ignoring what eslint wants. That change is approved by eslint in  https://github.com/openSUSE/cockpit-tukit/pull/58 and at least on my machine, running `npm run eslint:fix` doesn't change it. @SludgeGirl could you please check that too?